Gemfile.lock

```diff
@@ -662,7 +677,7 @@ GEM
     xml-simple (1.1.4)
     xpath (2.0.0)
       nokogiri (~> 1.3)
-    zeus (0.15.4)
+    zeus (0.15.13)
```

Comment:
This version messes up my terminal: https://github.com/burke/zeus/issues/624
Version 0.15.14 works fine for me.